### PR TITLE
Email-notification for sage intacct app changes commit

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/core/models/schedule-setting.model.ts [hunk 1]
```diff
@@ -7,4 +7,6 @@ export type ScheduleSettings = {
   start_datetime: Date;
   interval_hours: number;
   schedule: number;
+  emails_selected: [];
```
Comment (ashwin1111) [inline src/app/core/models/schedule-setting.model.ts:10]: string[];

File: src/app/core/models/schedule-setting.model.ts [hunk 2]
```diff
@@ -7,4 +7,6 @@ export type ScheduleSettings = {
...
+  additional_email_options:[];
```
Comment (ashwin1111) [inline src/app/core/models/schedule-setting.model.ts:11]: string[];

File: src/app/core/models/schedule-setting.model.ts (hunk 1 above)
Comment (ashwin1111) [reply src/app/core/models/schedule-setting.model.ts:10]: or whatever the db column type is

File: src/app/core/models/schedule-setting.model.ts (hunk 2 above)
Comment (ashwin1111) [reply src/app/core/models/schedule-setting.model.ts:11]: or whatever the db column type is

File: src/app/shared/simple-search-select/simple-search-select.component.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<form [formGroup]="form" class="search-select">
+    <div class="search-select--search-input">
+        <div fxLayout="row" class="search-select--search-area">
+            <img class="search-select--search-icon" src="assets/images/pngs/search.svg">
+            <input class="search-select--input-field" placeholder="Search By Name"
+                formControlName="searchOption" cdkFocusInitial type="text" matInput>
+        </div>
+    </div>
+</form>
```
Comment (ashwin1111) [inline src/app/shared/simple-search-select/simple-search-select.component.html:9]: new line at the end of file

File: src/app/shared/simple-search-select/simple-search-select.component.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.search-select {
+    background: #f5f5f5;
+
+    &--search-input {
+        padding: 7px 9px 7px 9px;
+    //     display: flex;
```
Comment (ashwin1111) [inline src/app/shared/simple-search-select/simple-search-select.component.scss:6]: remove dead code

File: src/app/si/settings/schedule/schedule.component.html [hunk 5]
```diff
@@ -2,19 +2,135 @@
...
+             matTooltip="The selected email address will be notified instantly upon export failure"
+             [inline]="true" 
+             matTooltipPosition="after"
+             >info</mat-icon>
+          </div>
+          <div class="schedule--header-info">
+            <div fxLayout="row">
+              You can select one or more email address from the given list,
+               or add new one to get email notifications
+              <a href="https://help.fylehq.com/en/articles/6064944-how-to-be-notified-when-auto-export-fails-in-fyle-netsuite-integration" target="_blank"
```
Comment (ashwin1111) [inline src/app/si/settings/schedule/schedule.component.html:65]: kb article link has to be updated

File: src/app/si/settings/schedule/add-email-dialog/add-email-dialog.component.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    <div class="mat-title">
+      Add new Email Address
+    </div>
+    <mat-icon class="cursor-pointer" aria-hidden="false" aria-label="close this dialog" mat-dialog-close>close
+    </mat-icon>
+  </div>
+  <hr />
+</h2>
+<div class="admin-info">
+  Add an email address on which you would like to recieve your integration notifcation mails.
```
Comment (ashwin1111) [inline src/app/si/settings/schedule/add-email-dialog/add-email-dialog.component.html:12]: `notification` typo

File: src/app/si/settings/schedule/add-email-dialog/add-email-dialog.component.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        </mat-form-field>
+      </div>
+    </form>
+  </mat-dialog-content>
+  <mat-dialog-actions align="end">
+    <button type="button" mat-flat-button mat-dialog-close>Cancel</button>
+    <button [disabled]="!form.valid" class="mappings-dialog--save-btn" mat-flat-button color="primary" type="submit" (click)="submit()">
+      Add and Save
+    </button>
+  </mat-dialog-actions>
```
Comment (ashwin1111) [inline src/app/si/settings/schedule/add-email-dialog/add-email-dialog.component.html:44]: eof

File: src/app/si/settings/schedule/schedule.component.html [hunk 8]
```diff
@@ -2,19 +2,135 @@
   <app-loader></app-loader>
 </div>
 <div *ngIf="!isLoading">
-  <div class="page-header schedule--header">
+  <!-- <div class="page-header schedule--header"> -->
```
Comment (ashwin1111) [inline src/app/si/settings/schedule/schedule.component.html:5]: remove dead code

File: src/app/si/settings/schedule/schedule.component.html [hunk 9]
```diff
@@ -2,19 +2,135 @@
...
+    </div>
+
     <div fxLayout="column" fxLayoutAlign="start stretch">
+      <div *ngIf="form.controls.scheduleEnabled.value">
+        <div fxLayout="column" class="schedule--select" fxLayoutAlign="start stretch">
+          <div fxLayout="row" class="schedule--header-title" fxLayoutAlign="start center">
+            Select the frequency of the export
+            <mandatory-field></mandatory-field>
+          </div>
+          <span class="schedule--header-info">Select the frequancy at which you want your Automated import-export to
```
Comment (ashwin1111) [inline src/app/si/settings/schedule/schedule.component.html:38]: `frequency` typo

File: src/app/si/settings/schedule/schedule.component.ts [hunk 10]
```diff
@@ -18,35 +21,82 @@ export class ScheduleComponent implements OnInit {
   isLoading = false;
   hours = [...Array(24).keys()].map(day => day + 1);
   settings: ScheduleSettings;
+  workspaceAdmins = [];
```
Comment (ashwin1111) [inline src/app/si/settings/schedule/schedule.component.ts:24]: mention type

**Review by ashwin1111 — CHANGES_REQUESTED** [review]

File: src/app/si/settings/schedule/schedule.component.html (hunk 5 above)
Comment (DhaaraniCIT) [reply src/app/si/settings/schedule/schedule.component.html:65]: what is the link

Comment (NileshPant1999) [reply src/app/si/settings/schedule/schedule.component.html:62]: for now you can comment this out Gayathiri will create a new one and will give you the link.

File: src/app/core/models/schedule-setting.model.ts [hunk 11]
```diff
@@ -7,4 +7,6 @@ export type ScheduleSettings = {
   start_datetime: Date;
   interval_hours: number;
   schedule: number;
+  emails_selected: string[];
+  additional_email_options: any[];
```
Comment (NileshPant1999) [inline src/app/core/models/schedule-setting.model.ts:11]: why any ??

File: src/app/si/settings/schedule/schedule.component.ts [hunk 12]
```diff
@@ -18,35 +21,82 @@ export class ScheduleComponent implements OnInit {
   isLoading = false;
   hours = [...Array(24).keys()].map(day => day + 1);
   settings: ScheduleSettings;
+  workspaceAdmins: any[] = [];
```
Comment (NileshPant1999) [inline src/app/si/settings/schedule/schedule.component.ts:24]: why any ??

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**